packages/ncursesw.rb

```diff
@@ -46,5 +46,12 @@ def self.install
     Dir.chdir 'ncursesw_build' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     end
+    # Make symlinks for non-wide software
+    Dir.chdir "#{CREW_DEST_LIB_PREFIX}" do
+      Dir.glob('*w.so*').each do |f|
+        @basefile_nowide = f.gsub('w.so', '.so')
+        FileUtils.ln_sf f, @basefile_nowide
+      end
+    end
```

Comment:
This is quite the hack.  Not sure this is the best approach.  Are you doing this to avoid having to recompile packages?  If that is the case then what happens if a package is updated and needs to be recompiled?  Will it still work as expected?

Reply:
This is working in CREW_DEST so only changing things for output libraries.

But this is a `ncurses` issue where the wide libraries are named with a `w`.

Yes this is to avoid recompiling packages, but also EVERYTHING breaks otherwise. It's just a complete shitshow. Building is hell.